pkgtree/pkgtree.go

```diff
@@ -0,0 +1,898 @@
+package pkgtree
+
+import (
+	"fmt"
+	"go/build"
+	"go/parser"
+	gscan "go/scanner"
+	"go/token"
+	"os"
+	"path/filepath"
+	"sort"
+	"strconv"
+	"strings"
+	"unicode"
+)
+
+// Stored as a var so that tests can swap it out. Ugh globals, ugh.
+var IsStdLib = doIsStdLib
```

Comment:
This is one of those things I'd really strongly prefer not to export, but we have the call to it in the solver.

If we can't think of any clever solutions, then I'd prefer we duplicate the function in both packages than have it exposed. The only solution I can think of is adding a stdlib filtering param to `ToReachMap()`...but that doesn't take care of the calls in `rootdata.go`.